circle.yml

```diff
@@ -0,0 +1,13 @@
+machine:
+  node:
+    version: 6.9
+  environment:
+    PATH: "${PATH}:${HOME}/${CIRCLE_PROJECT_REPONAME}/node_modules/.bin"
+dependencies:
+  override:
+    - yarn
+  cache_directories:
+    - ~/.cache/yarn
```

Comment:
I pretty much just copy/pasted this config from the circle docs: https://circleci.com/docs/1.0/yarn/